mesh/src/main/java/no/nordicsemi/android/mesh/sensorutils/ElectricCurrent.java

```diff
@@ -0,0 +1,52 @@
+package no.nordicsemi.android.mesh.sensorutils;
+
+import static no.nordicsemi.android.mesh.utils.MeshParserUtils.unsignedBytesToInt;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.RestrictTo;
+
+/**
+ * The Electric Current characteristic is used to represent a measure of current in units of ampere.
+ */
+public class ElectricCurrent extends DevicePropertyCharacteristic<Float> {
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY)
+    public ElectricCurrent(@NonNull final byte[] data, final int offset, final int length) {
+        super(data, offset, length);
+        int bits = unsignedBytesToInt(data[offset], data[offset + 1]);
```

Comment:
Cool, a third way of doing this :)

I guess the little/big endian is correct?

Reply:
Yes, looking and source and running tests tells you so :)